src/wp-includes/script-loader.php

```diff
@@ -2404,6 +2404,7 @@ function wp_global_styles_render_svg_filters() {
 	 */
 	if (
 		is_admin() &&
+		get_current_screen() !== null &&
 		! get_current_screen()->is_block_editor()
```

Comment:
Rather than call `get_current_screen()` twice, can we create a `$screen` variable at the top of the function (above the multiline comment, with a line break), and have both of these conditions use the `$screen` variable instead?

Also, for the new condition, can we replace this with `$screen instanceof WP_Screen`? This will ensure that it's a `WP_Screen` object before attempting to call the `::is_block_editor()` function in the last condition.

Reply:
Actually, hold off on the above suggestion for now. Pinging @SergeyBiryukov for a second opinion on this.

Sergey, some files in Core have multiple calls to `get_current_screen()`. Normally, we'd avoid multiple function calls. Is this something we should avoid here, or is this an exception?

Reply:
Looking at the contents of the [`get_current_screen()`](https://developer.wordpress.org/reference/functions/get_current_screen/) function, @costdev's suggestions make sense to me.

I'd also would like to see a test covering the originally reported issue (throwing an error on PHP 8.1 without the fix and passing with the fix).

Reply:
Regarding the test: The function relies on `is_admin()`, which, based on [its source](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/load.php#L1173) will require `WP_ADMIN` to be defined in order to test this issue and fix.

`WP_ADMIN` isn't defined in the test suite, and doing so in this test would impact other tests (including for this function). Not sure how to proceed with that. 

That said, here's two screenshots:

As-is:
![image](https://user-images.githubusercontent.com/79332690/188635072-8406787e-bd92-4ec5-a25e-369eacc80fb7.png)

With new condition:
![image](https://user-images.githubusercontent.com/79332690/188637796-2126afa2-87c9-4c00-a602-79dcfbae4926.png)

Reply:
Yes, you got a point about the `define()`. There are ways around it (run in separate process, run as separate test suite), but those are outside the scope of this particular fix.